docs/admin/resourcequota/index.md

```diff
@@ -63,6 +63,7 @@ in a namespace can be limited.  The following compute resource types are support
 | `memory` | Across all pods in a non-terminal state, the sum of memory requests cannot exceed this value. |
 | `requests.cpu` | Across all pods in a non-terminal state, the sum of CPU requests cannot exceed this value. |
 | `requests.memory` | Across all pods in a non-terminal state, the sum of memory requests cannot exceed this value. |
+| `requests.storage` | Across all persistent volume claims, the sum of storage requests cannot exceed this value. |
```

Comment:
do you mind splitting this into a separate PR?  this made kube 1.4, but the rest will not make kube 1.4 given feature freeze.